spec/models/redirection_spec.rb

```diff
@@ -83,4 +83,32 @@
       expect(redirection.tag_uri).to eq(id)
     end
   end
+
+  describe "#unlink" do
+    it "destroys the redirection" do
+      redirection = create(:redirection)
+
+      redirection.unlink
+
+      expect(Redirection.find_by(id: redirection.id)).to be_nil
```

Comment:
Could also do:

```ruby
redirection_count = Redirection.count

# do stuff

expect(Redirection.count).to eq(redirection_count - 1)
```

I kind of like your nil check though. What about something like:

```ruby
redirection = create(:redirection)

redirection.unlink
redirection.reload

expect(redirection).to be_nil
```

Yet another alternative could be a `deleted?` predicate on `Redirection` and then we could use `be_deleted`